ledger/src/bank_forks_utils.rs

```diff
@@ -21,14 +21,18 @@ pub type LoadResult = result::Result<
 >;
 
 fn to_loadresult(
-    brp: BlockstoreProcessorResult,
-    snapshot_hash: Option<(Slot, Hash)>,
+    bpr: BlockstoreProcessorResult,
+    snapshot_slot_and_hash: Option<(Slot, Hash)>,
 ) -> LoadResult {
-    brp.map(|(bank_forks, leader_schedule_cache)| {
-        (bank_forks, leader_schedule_cache, snapshot_hash)
+    bpr.map(|(bank_forks, leader_schedule_cache)| {
+        (bank_forks, leader_schedule_cache, snapshot_slot_and_hash)
     })
 }
 
+/// Load the banks and accounts
+///
+/// If a snapshot config is given, and a snapshot is found, it will be loaded.  Otherwise, load
+/// from genesis.
 pub fn load(
```

Comment:
I refactored `load()` to make it more clear if we are loading from a snapshot or from genesis.